src/main/java/passableleaves/core/PLCore.java

```diff
@@ -27,7 +29,7 @@ public PLCore() {
 
     @Override
     public String[] getASMTransformerClass() {
-        return new String[0];
+        return new String[] {"passableleaves.core.PLTransformer"};
```

Comment:
They're only passed obfuscated names if they run before the FMLRemapper (sorting index <1000). Since the sorting index of this core mod is 1001, the transformer received srg names. This transformer is an ASM transformer, not an Access Transformer anyway.